ui/raidboss/data/07-dt/raid/r6s.ts

```diff
@@ -159,20 +159,23 @@ const triggerSet: TriggerSet<Data> = {
       outputStrings: {
         avoidCleave: {
           en: 'Be on boss hitbox (avoid tank cleaves)',
+          de: 'Geh auf den Kreis vom Boss (vermeide Tank-Cleaves)',
           fr: 'Sur la hitbox (évitez les tanks cleaves)',
           ja: 'ボス背面のサークル上に',
           cn: '站在目标圈上 (远离坦克死刑)',
           ko: '보스 히트박스 경계에 있기 (광역 탱버 피하기)',
         },
         warmCleave: {
           en: 'Tank cleave on YOU (${dir} => get hit by Red)',
+          de: 'Tank-Cleave auf DIR (${dir} => von Rot treffen lassen)',
           fr: 'Tank cleave sur VOUS (${dir} => prenez le Rouge)',
           ja: 'タンク攻撃 (${dir} => 赤に当たる)',
           cn: '坦克死刑 (${dir} => 吃红色)',
           ko: '광역 탱버 대상자 (${dir} => 빨간색 맞기)',
         },
         coolCleave: {
           en: 'Tank cleave on YOU (${dir} => get hit by Blue)',
+          de: 'Tank-Cleave auf DIR (${dir} => von Blau treffen lassen)',
           fr: 'Tank cleave sur VOUS (${dir} => prenez le Bleu)',
           ja: 'タンク攻撃 (${dir} => 青に当たる)',
           cn: '坦克死刑 (${dir} => 吃蓝色)',
```

Comment:
We don't use hyphens in other tanklibs, so why not here?

https://github.com/OverlayPlugin/cactbot/blob/e82e40900c37543b84c670cecf0cd609e43a146d/resources/outputs.ts#L105-L136